### ISSUE TITLE
Error: Cannot find module 'yeoman-generator/node_modules/path-exists'

### ISSUE THREAD

**Issue by konstantinzolotarev** [trailsjs/generator-trails#18]
Copied from trailsjs/trails#53. Thank you @muxahuk
trying generate controller gives the same error:
```
C:\Users\M7\trails-demo2>yo trails:controller Main
module.js:328
    throw err;
    ^

Error: Cannot find module 'yeoman-generator/node_modules/path-exists'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (C:\Users\M7\AppData\Roaming\npm\node_modules\generator-trails\generators\app\util.js:13:46)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
```

**Comment by jaumard** [comment]
For the record, fix on https://github.com/trailsjs/generator-trails/pull/16

**Comment by jaumard** [comment]
Fix on v0.0.10
